bootloader/bootloader.yul

```diff
@@ -3,6 +3,7 @@ object "Bootloader" {
     }
     object "Bootloader_deployed" {
         code {
+            {{CODE_START_PLACEHOLDER}}
```

Comment:
Just in case, in the current form it will most likely change the bootloader hash, while the empty line created here does not change the logic, the bytecode contains the metadata hash which includes the hash of the source code

Reply:
ack - will merge it in 'dev' branch